cli/internal/cache/cache_http_test.go

```diff
@@ -178,17 +179,20 @@ func TestRestoreTar(t *testing.T) {
 
 	tar := makeValidTar(t)
 
-	expectedFiles := []string{
-		"extra-file",
-		"my-pkg/",
-		"my-pkg/some-file",
-		"my-pkg/link-to-extra-file",
-		"my-pkg/broken-link",
+	expectedFiles := []turbopath.AnchoredSystemPath{
+		turbopath.AnchoredUnixPath("extra-file").ToSystemPath(),
+		turbopath.AnchoredUnixPath("my-pkg/").ToSystemPath(),
+		turbopath.AnchoredUnixPath("my-pkg/some-file").ToSystemPath(),
+		turbopath.AnchoredUnixPath("my-pkg/link-to-extra-file").ToSystemPath(),
+		turbopath.AnchoredUnixPath("my-pkg/broken-link").ToSystemPath(),
 	}
 	files, err := restoreTar(root, tar)
 	assert.NilError(t, err, "readTar")
 
-	expectedSet := util.SetFromStrings(expectedFiles)
+	expectedSet := make(util.Set)
```

Comment:
Previously `restoreTar` was _always_ returning Unix-separated paths, even on Windows. However, downstream uses were treating these as system-separated paths. Likely didn't manifest in user-facing issues because Windows supports `/` as a separator, but no guarantees.